core/src/main/scala/org/apache/spark/deploy/worker/ExternalShuffleService.scala

```diff
@@ -18,12 +18,15 @@
 package org.apache.spark.deploy.worker
```

Comment:
This is in the wrong package. It's no longer used for standalone mode only. It might make sense to put it in `o.a.s.shuffle`